concent_api/core/tests/test_payments_bankster.py

```diff
@@ -411,38 +401,52 @@ def test_that_settle_overdue_acceptances_should_return_claim_deposit_with_amount
+        with freeze_time("2018-02-04 10:00:15"):
+            subtask_results_accepted_list = [
+                self._get_deserialized_subtask_results_accepted(
+                    report_computed_task=self._get_deserialized_report_computed_task(
+                        timestamp="2018-02-04 9:00:05",
+                        task_to_compute=task_to_compute
+                    ),
+                    payment_ts="2018-02-04 10:00:05",
                 ),
-                payment_ts="2018-02-05 12:00:00",
-            )
-        ]
-
-        with mock.patch(
-            'core.payments.bankster.service.get_list_of_payments',
-            side_effect=[
-                [self._create_batch_payment_object(amount=1000, closure_time=subtask_results_accepted_list[0].payment_ts + 1)],
-                self._get_list_of_force_transactions(),
-            ]
-        ) as get_list_of_payments_mock:
-            with self.assertRaises(BanksterTimestampError):
-                settle_overdue_acceptances(
-                    requestor_ethereum_address=task_to_compute.requestor_ethereum_address,
-                    provider_ethereum_address=task_to_compute.provider_ethereum_address,
-                    acceptances=subtask_results_accepted_list,
-                    requestor_public_key=hex_to_bytes_convert(task_to_compute.requestor_public_key),
+                self._get_deserialized_subtask_results_accepted(
+                    report_computed_task=self._get_deserialized_report_computed_task(
+                        timestamp="2018-02-04 23:00:05",
+                        task_to_compute=task_to_compute
+                    ),
+                    payment_ts="2018-02-05 10:00:05",
                 )
+            ]
 
+        with freeze_time("2018-02-05 10:00:25"):
+            with mock.patch('core.payments.bankster.service.get_deposit_value', return_value=5000) as get_deposit_value_mock:
+                with mock.patch(
+                    'core.payments.bankster.service.get_list_of_payments',
+                    side_effect=[
+                        self._get_list_of_batch_transactions(),
+                        self._get_list_of_force_transactions(),
+                    ]
+                ) as get_list_of_payments_mock:
+                    claim_against_requestor = settle_overdue_acceptances(
+                        requestor_ethereum_address=task_to_compute.requestor_ethereum_address,
+                        provider_ethereum_address=task_to_compute.provider_ethereum_address,
+                        acceptances=subtask_results_accepted_list,
+                        requestor_public_key=hex_to_bytes_convert(task_to_compute.requestor_public_key),
+                    )
+
+        get_deposit_value_mock.assert_called_once()
         get_list_of_payments_mock.assert_called()
 
+        self.assertIsNotNone(claim_against_requestor.tx_hash)
+
+        self.assertEqual(claim_against_requestor.amount, 2000)
```

Comment:
Add comment how this value was calculated.

Reply:
Done